Automattic/wp-calypso
# Stats: Removing hard-coded tiers

Related to #84534

## Proposed Changes

* updating tier slider with values and prices from the new endpoint
* adding a popover explaining the highest tier pricing and extension

## Testing Instructions



* navigate to a live branch
* apply the feature flag `stats/tier-upgrade-slider`
* in case you are viewing stats for a non-commercial website click on the link allowing the purchase of a commercial licence
* check various pricing options and verify that they display prices in your currency
* check that the highest tier also shows a popover with additional information

After the changes:
<img width="690" alt="SCR-20231129-nzix" src="https://github.com/Automattic/wp-calypso/assets/112354940/c0514337-a3cb-4dcd-8385-b9092bc54fd4">



## Pre-merge Checklist
